docs/import-and-export.md

```diff
@@ -0,0 +1,28 @@
+<!--
+@TODO: Update this for v2.5
+-->
+
+## Export to files
+
+You can specify the output file of your exported journal using the
+`-o` argument
```

Comment:
This isn't true anymore. `-o` will still work, but only for backward compatibility. They should use `--file` like in `formats.md`.

Reply:
Actually, do we still need this file? Maybe `formats.md` is enough to cover this?